src/main/java/com/prgms/vouchermanager/repository/voucher/FileVoucherRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.prgms.vouchermanager.repository.voucher;
+
+
+import com.prgms.vouchermanager.domain.voucher.Voucher;
+import com.prgms.vouchermanager.util.file.FileManager;
+import jakarta.annotation.PreDestroy;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Repository
+@Profile("file")
+public class FileVoucherRepository implements VoucherRepository{
+    private final FileManager fileManager;
+
+    private Map<UUID,Voucher>vouchers ;
+
+    public FileVoucherRepository(FileManager fileManager) {
+        this.fileManager = fileManager;
+        vouchers = fileManager.readVoucherCsv();
+
+
+    }
```

Comment:
```suggestion
    private final Map<UUID, Voucher> vouchers;

    public FileVoucherRepository(FileManager fileManager) {
        this.fileManager = fileManager;
        vouchers = fileManager.readVoucherCsv();
    }
```

fileManager 로 초기화 한다면 vouchers 는 final로 변경하는 것이 좋겠네요.!